### ISSUE TITLE
Application crashing

### ISSUE THREAD

**Issue by davad** [KyleAMathews/Journal#7]
I pulled the latest, and it's crashing now. I haven't had a chance to dig through it yet, but here's the traceback when I start it locally with `foreman`.

```
[0] % foreman start     
09:44:22 web.1  | started with pid 16966
09:44:22 web.1  | TypeError: Object #<Schema> has no method 'get'
09:44:22 web.1  |   at Generator.generateMapping (Journal/node_modules/mongoosastic/lib/mapping-generator.js:6:27)
09:44:22 web.1  |   at createMappingIfNotPresent (Journal/node_modules/mongoosastic/lib/mongoosastic.js:142:13)
09:44:22 web.1  |   at Function.module.exports.schema.statics.createMapping (Journal/node_modules/mongoosastic/lib/mongoosastic.js:39:5)
09:44:22 web.1  |   at Object.<anonymous> (Journal/post_schema.coffee:75:8)
09:44:22 web.1  |   at Object.<anonymous> (Journal/post_schema.coffee:128:4)
09:44:22 web.1  |   at Module._compile (module.js:449:26)
09:44:22 web.1  |   at Object.loadFile (Journal/node_modules/coffee-script-brunch/node_modules/coffee-script/lib/coffee-script/coffee-script.js:179:19)
09:44:22 web.1  |   at Module.load (module.js:356:32)
09:44:22 web.1  |   at Function.Module._load (module.js:312:12)
09:44:22 web.1  |   at Module.require (module.js:362:17)
09:44:22 web.1  |   at require (module.js:378:17)
09:44:22 web.1  |   at Object.<anonymous> (Journal/server.coffee:7:1, <js>:14:3)
09:44:22 web.1  |   at Object.<anonymous> (Journal/server.coffee:574:4)
09:44:22 web.1  |   at Module._compile (module.js:449:26)
09:44:22 web.1  | exited with code 1
09:44:22 system | sending SIGTERM to all processes 
SIGTERM received
```

On Heroku, I get this error.
    `2013-04-09T15:48:18.728779+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path=/posts?created=2013-04-09T15%3A48%3A56.216Z host=journal.dmwl.net fwd="98.202.243.63" dyno= connect= service= status=503 bytes=`

**Comment by KyleAMathews** [comment]
Huh, yeah try to narrow that down to which line of code is causing the problem. I'm not seeing any issues.

**Comment by davad** [comment]
So it's having a problem with post_schema.coffee:25, the createMapping method. _Why_ it's suddenly having a problem, I have no idea. But I compiled and ran all the code, and that was the most specific line in the Journal app with a problem.

The only difference between the code I'm using and the one you're using is the configuration stuff. I tried switching ElasticSearch to static values (since that was just a few lines ahead of this), but that didn't make a difference. Any ideas?

**Comment by KyleAMathews** [comment]
Hmmm... you want to try commenting out https://github.com/KyleAMathews/Journal/blob/master/post_schema.coffee#L25 and see if that fixes things?

**Comment by davad** [comment]
Yup, it runs without error now. What is that mapping supposed to be doing?

**Comment by KyleAMathews** [comment]
That's what activates the ElasticSearch indexing.

**Comment by davad** [comment]
Looks like this was fixed in f4be2404f80512233ff6149bca6b396cdeb9c7d5
